db/schema.rb

```diff
@@ -333,11 +333,11 @@
 
   create_table "hesa_collection_requests", force: :cascade do |t|
     t.string "collection_reference"
+    t.datetime "requested_at"
```

Comment:
this line was changed due to running a migration

Reply:
I think it's best to generally not commit these when there isn't actually a change to the database. I get them sometimes too where the column order changes. Never worked out why but if you commit it and the next persons migration puts it back in the previous order we'll be committing them for ever more.